doc/voq/fabric.md

```diff
@@ -171,6 +172,85 @@ PORT            RxCells     TxCells      Crc       Fec  Corrected
 
 In a later phase, a `show fabric status` command will be added to show the remote switch ID and link ID for each fabric link of an ASIC. This will be obtained from the SAI_PORT_ATTR_FABRIC_REACHABILITY port attribute of the fabric port. Note that for fabric links that do not have a link partner because of the configuration of the chassis, this will show the status as `down`. The status will also be `down` for fabric links that are down due to some other physical error. To identify links that are down due to error vs links that are not expected to be up because of the chassis connectivity, we need to build up a list of expected fabric connectivity for each ASIC. This can be computed ahead of time based on the vendor configuration and populated in the minigraph. This will be implemented in a later phase.
 
+## 2.8 Fabric Link Monitor
+
+SONiC needs to monitor the fabric link status and take corresponding actions once an unhealthy link is detected to avoid traffic loss. Once the fabric link monitoring feature is enabled, SONiC needs to monitor the fabric capacity of a forwarding ASIC and take corresponding action once the capacity goes below the configured threshold.
+
+The design of fabric link monitor is intentionally scoped to use local component state such as information local to a linecard or information local to a supervisor. This design simplifies the need for inter-component communication.
+
+### 2.8.1 Monitor Fabric Link Status
+
+Unhealthy fabric links may lead to traffic drops. Fabric link monitoring is an important tool to minimize traffic loss. The fabric link monitor algorithm monitors fabric link status and takes links out of service if one or more criteria are true. This is needed on both fabric ASICs and forwarding ASICs.
+
+#### 2.8.1.1 Cli commands
+
+```
+> config fabric port set [port_id] isolate
```

Comment:
can you change the command `config fabric port isolate <port_id>`

Reply:
I updated document  . thank you